webpack.config.js

```diff
@@ -30,6 +31,7 @@ module.exports = {
   plugins: [
     new MiniCssExtractPlugin(),
     process.env.REPLAY_BUILD_VISUALIZE && new Visualizer(),
+    new webpack.EnvironmentPlugin({ REPLAY_RELEASE: undefined }),
```

Comment:
should the release by undefined?

Reply:
`undefined` is just the default value that will be used if `REPLAY_RELEASE` is not defined in `process.env`. However, maybe we should use `"development"` as the default value (which is also used as the default in another place in this PR).

Reply:
According to the webpack docs if you make it undefined Webpack yells at you when you don't define it, which I thought was a nice side-effect. We handle it being falsey in the code and default to "development" there.